src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -59,8 +59,7 @@ private LocalDefinition EmitAddress(BoundExpression expression, AddressKind addr
                     break;
 
                 case BoundKind.Parameter:
-                    EmitParameterAddress((BoundParameter)expression);
-                    break;
+                    return EmitParameterAddress((BoundParameter)expression, addressKind);
```

Comment:
It looks like you fixed a bug here. What scenario was affected?

Reply:
We could have situations when `ref readonly` parameter could be mutated. See test `InParamGenericReadonly` in `CodegenInparametersTests` as a specific example.

---
In reply to: [139487394](https://github.com/dotnet/roslyn/pull/21982#discussion_r139487394) [](ancestors = 139487394)